website/server/controllers/api-v3/groups.js

```diff
@@ -1068,6 +1069,36 @@ api.inviteToGroup = {
 
     Group.validateInvitations(uuids, emails, res);
 
+    // If party, check the limit of members
+    if (group.type === 'party') {
+      let memberCount = 0;
+
+      // Counting the members that already joined the party
+      memberCount += group.memberCount;
+
+      // Count how many invitations currently exist in the party
+      let query = {};
+      query['invitations.party.id'] = group._id;
+      let groupInvites = await User
+        .find(query)
+        .select('_id')
+        .exec();
+      memberCount += groupInvites.length;
+      
+      // Counting the members that are going to be invited by email and uuids
+      if (uuids) {
```

Comment:
last edit then I think it's good to go: since we already have a function to validate invites ( `Group.validateInvitations` ) I think we should use that for validating the number of invites too. It's only used here so feel free to change it if you need to pass any other parameter that you might need